cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/CouchDbException.java

```diff
@@ -76,7 +82,17 @@ public String getReason() {
     public String getMessage() {
         //improve the message with status code, error and reason if we have them
         return ((getStatusCode() > 0) ? getStatusCode() + " " : "") + super.getMessage()
+                + ((url != null) ? ": " + url : "")
```

Comment:
+1 for the human readable `404 Object Not Found at https://examples.cloudant.com/mydb. Error: not_found. Reason: no_db_file`.